python/ray/tune/test/dependency_test.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python
```

Comment:
This is fine if this is the only thing that's ever going to be in this test file. However in general new tests should all use `pytest`.

Reply:
Yeah this one kinda has to be its own file since it tests import state.